wordpress-mobile/WordPress-iOS
# Plugin directory search

Adding search to Plugin Directory!

To test:

1. Open plugin directory
2. Tap on the "search icon" in the top right corner
3. Type a plugin you're looking for
4. Verify that there are results returned
5.  (Optional) Install the plugin!

Two notes: the search algorithm itself isn't... great. For example, it returns a bunch of results for `guten`, but none for `gutenb` or `gutenbe`, and than one (and _not_ the "real" Gutenberg) for `gutenber`. Not much we can do about that.

This also desperately needs an empty state for when the search results comes back empty — it'll come in a separate PR, with more empty states for Plugin Directory.
